### ISSUE TITLE
HTML Graph Rendering Error

### ISSUE THREAD

**Issue by 0FiRE0** [allinurl/goaccess#1570]
The generated HTML file is causing an error in the browser while rendering the page, which causes all the graphs to disappear.
The error is caused by extra 2 characters appended at the end of the script which are "j!" before closing the "</script>" tag.

Issue appearance:
![Goaccess Issue](https://user-images.githubusercontent.com/19993814/66873748-eb73de80-efb1-11e9-9d8d-9118561ae374.PNG)

Extra characters below:
![Goaccess Issue](https://user-images.githubusercontent.com/19993814/66873306-caf75480-efb0-11e9-9d38-19f452b5b5a4.PNG)
.

Once these 2 characters are removed manually from the generated HTML file, the browsers will be able to show back the graphs in the page properly.

**Comment by allinurl** [comment]
Thanks for reporting this. I don't see any `j!` characters on the [generated report](view-source:https://rt.goaccess.io/?20190828082924). Are you using a different char encoding? Does it happen when parsing only a few lines, e.g., 5 lines? Thanks

**Comment by 0FiRE0** [comment]
The character encoding had not been changed, it used to work just fine, and this issue appeared after updating from the verified commit on "13 Apr 2019" (where everything was fine) to the latest verified commit.

The parsed nginx log was more than 1,000 lines.

Please note that I am using the Docker image, and changing the Dockerfile by commenting the `CMD ["--help"]` line because it is causing the container to restart infinity without being able to use it at all.

**Comment by welchr** [comment]
I was having this same issue as well, but updated to the latest docker image and it seems to be resolved. I'm using: 

```bash
> sudo docker image ls --digests allinurl/goaccess
REPOSITORY          TAG                 DIGEST                                                                    IMAGE ID            CREATED             SIZE
allinurl/goaccess   latest              sha256:cee855f2e8022dca04c72df14e73c42f04a3253655db25634ff21ee8721caa50   e14f6da00309        2 months ago        13.8MB
```

Maybe it was fixed in 9271b97c4e1bdcad143e9263e9f1e89dde5fa5e5?

**Comment by allinurl** [comment]
@welchr is right. @0FiRE0 please try building from [**development**](https://github.com/allinurl/goaccess#build-from-github-development) as I think this was already fixed upstream.

**Comment by allinurl** [comment]
This should be fixed per #1595.

Closing it, feel free to reopen it if needed.

**Comment by 0FiRE0** [comment]
Thank you @allinurl 
I have pulled the latest commit from the `master` branch and re-built it again, and it solved the issue.
